src/merfin/merfin.C

```diff
@@ -355,8 +354,8 @@ histKmetric(char               *outName,
   
   fprintf(stderr, "\nNumber of contigs: %u\n", ctgn);
 
-
-#pragma omp parallel for reduction (+:overcpy,tot_missing,tot_kasm) num_threads(threads) schedule(dynamic)
```

Comment:
Since the `stderr` output is no longer ordered, printing the total missing value after each contig isn't as meaningful. It would be possible to add `missing` to the reduction and simplify some other lines if that was no longer be printed.

Reply:
Hi, Alex- I'm in the middle of replacing the current threading model with one that operates with an input/worker/output pattern.  One thread reads input, passes that to N worker threads, and those pass output to one output thread.  That will result in good balancing and allow true progress reporting.

I'll leave this open and use it for comparison.  If this gets merged in, it'll probably end up on a branch.